frontend/src/Utilities/String/translate.ts

```diff
@@ -25,7 +25,7 @@ export async function fetchTranslations(): Promise<boolean> {
 
 export default function translate(
   key: string,
-  tokens?: Record<string, string | number | boolean>
+  tokens?: Record<string, string | number | boolean | undefined>
```

Comment:
I think overall it'd be better to have the caller set a default value if it's undefined, so it can properly deal with the differences, otherwise the translation is going to look off.

Reply:
reverted change